oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileNodeStoreBuilder.java

```diff
@@ -59,12 +59,30 @@
     /**
      * Default value for {@link #PROP_THREAD_POOL_SIZE}
      */
-    static final String DEFAULT_NUMBER_OF_DATA_DUMP_THREADS = "4";
+    static final int DEFAULT_NUMBER_OF_DATA_DUMP_THREADS = 4;
     /**
      * System property for specifying number of threads for parallel download when using {@link MultithreadedTraverseWithSortStrategy}
      */
     static final String PROP_THREAD_POOL_SIZE = "oak.indexer.dataDumpThreadPoolSize";
 
+    /**
+     * Default value for {@link #PROP_MERGE_THREAD_POOL_SIZE}
+     */
+    static final int DEFAULT_NUMBER_OF_MERGE_TASK_THREADS = 4;
```

Comment:
Sorry... I thought we only want _one_ thread that concurrently merges. Now we have 4... that would mean we can have 4 times 65 = 260 files open at the same time... I think 1 merge thread is more efficient.

Reply:
sure.